README.rst

```diff
@@ -19,14 +19,13 @@ Install
 
 ::
 
-  pip install Cython
   pip install embedded_jubatus
 
 Requirements
 ------------
 
-* Python 2.7, 3.3, 3.4 or 3.5.
-* `Jubatus <http://jubat.us/en/quickstart.html>`_ needs to be installed.
+* Python 2.7, 3.3, 3.4, 3.5, 3.6 or 3.7.
```

Comment:
Would you write `numpy` specified versions, from 1.14 to 1.16?

Reply:
Added description of all required libraries.

Reply:
Thanks a lot!